less/index.less

```diff
@@ -55,5 +55,171 @@ table {
 
 html, body {
     height: 100%;
-}
+}	
+
+
+html {
+	box-sizing: border-box;
+	font-size: 62.5%; 
+  }
+  
+  *, *:before, *:after {
+	box-sizing: inherit;
+  }
+  
+  h1, h2, h3, h4, h5 {
+	font-size: 2rem;
+  }
+  
+  img {
+   width: 100%;
+   height: auto;
+  }
+  
+  p {
+	line-height: 1.25;
+	margin: 20px 0;
+	font-size: 1.8rem;
+  }
+  
+  .container {
+	max-width: 600px;
+	margin: 0 auto;
+	font-size: 1.6rem;
+  }
+  
+  strong {
+	font-weight: bold;
+	color: red;
+  }
+  
+  header {
+	border-bottom: 1px solid black;
+	margin: 20px 0;
+	padding: 20px 0;
+	display: flex;
+	justify-content: space-between;
+  }
+  
+  @media( max-width: 500px ) {
```

Comment:
You should nest this media query inside of your header selector. Go through your code and see where you can nest me the media queries.